src/Sarif.Viewer.VisualStudio.Core/Models/ScrollViewerWrapper.cs

```diff
@@ -0,0 +1,126 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Media;
+
+using EnvDTE;
+
+using EnvDTE80;
+
+using Markdig.Wpf;
+
+using Microsoft.Sarif.Viewer.Tags;
+using Microsoft.VisualStudio.PlatformUI;
```

Comment:
Sorry, didn't notice earlier but some of these are not needed. Do a right-click > Remove and Sort Usings to get rid of them (including the "using Run =" which won't be needed anymore).

Reply:
I use this all the time to keep things clean. It even knows to do the grouping and put System.* at the top based on our StyleCop rules.